### PR TITLE
simple fix for attribute example file

### PR DESCRIPTION
**Pull request by commjoen**
simple fix for attribute example file

### CODE REVIEW

File: sample_attributes.yml [hunk 1]
```diff
@@ -4,9 +4,10 @@ registry_cert_path: /etc/docker/certs.d
 registry_name: /etc/docker/certs.d/registry_hostname:port
 registry_ca_file: /etc/docker/certs.d/registry_hostname:port/ca.crt
 container_user: vagrant
-container_capadd: null
+container_capadd: ''
```
Comment (chris-rock) [inline sample_attributes.yml]: Should we make this part of the inspec.yml? Then the benchmark would at least run out-of the box? e.g.

```
  - name: container_capadd
    required: true
    description: 'define needed capabilities for containers.'
    default: ''
    type: string
```

Comment (chris-rock) [reply sample_attributes.yml]: We could add a more detailed value like `NET_ADMIN,SYS_ADMIN` here

File: sample_attributes.yml [hunk 2]
```diff
@@ -4,9 +4,10 @@ registry_cert_path: /etc/docker/certs.d
...
 authorization_plugin: authz-broker
 log_driver: syslog
 log_opts: /syslog-address/
 app_armor_profile: docker-default
 selinux_profile: /label\:level\:s0-s0\:c1023/
+benchmark_version: 1.12.0
```
Comment (chris-rock) [inline sample_attributes.yml]: Should we make the a default too?

**Comment by chris-rock** [comment]
fixes #58

File: sample_attributes.yml (hunk 1 above)
Comment (commjoen) [reply sample_attributes.yml]: yes please :) , will do

File: sample_attributes.yml (hunk 2 above)
Comment (commjoen) [reply sample_attributes.yml]: will do so, closing this pr and creating a new one...

Comment (chris-rock) [reply sample_attributes.yml]: You can also force-push in your current branch and github updates the PR

Comment (commjoen) [reply sample_attributes.yml]: That did not worik... will create a new one with signoff

**Comment by commjoen** [comment]
See #61 with all the processed feedback as force pushing/updating the pr did not work for some weird reason.

**Pull request closed without merging**